static/js/biomedical/disease/types.ts

```diff
@@ -34,6 +36,16 @@ export interface DiseaseSymptomAssociationData {
   oddsRatio: number;
 }
 
+// specialization of NamedNode for disease parent data types
+export type DiseaseParent = NamedNode;
+
+export interface DiseaseParentTree {
```

Comment:
nit: maybe just `DiseaseTreeNode`

Reply:
Done!

Reply:
looks like you changed the name for the type in line 40 instead of here? & maybe for line 40, we can just do `NamedDisease` or not even give it a special name because technically it is still a named node.

Reply:
Sorry about that! The way GitHub PR review mode highlighted it seemed like. you wanted `DiseaseParent` name to change. You're right, its just a `NamedNode` so it doesn't need a specific name instantiation. So, I removed `DiseaseParent` and renamed `DiseaseParentTree` to `DiseaseTreeNodee`